develop/learning-paths/migration/articles/01-intro/01-from-a-monolith-to-a-modular-architecture.markdown

```diff
@@ -0,0 +1,124 @@
+of microservices, which we'll discuss next. 
+
+## The Microservices Concept
+
+The goal of a microservices is to do one (and only one) thing very
+well. Each microservice is small and stripped down of all but what it requires
+to meet its purpose. Their independent too. You can independently deploy each
+microservice and use it. But in addition to being independent, each microservice
+is composable, meaning it can be used with other microservices to do more
+complex things. 
+
+![Figure 1: Liferay drew inspiration from the concept of microservices. Their size, self reliance, and composability, make them ideal for a modular architecture.](../../images/microservice.png)
+
+In essence, we've rebuilt Liferay as microservices, to gain the ability to
+deploy new versions of the services in a few weeks, or even days, as compared
+previously to months. We've encapsulated these microservices in OSGi modules,
+which are a standards based deployment structure for deploying to dynamic Java
+systems.
+
+All of Liferay 7's modules can be developed and iterated on independently. Each
+module can have it's own release cycle. Developers can innovate rapidly over
+their modules. As with their modules, they can focus on providing the single
+purpose of their module.
+
+We'll go into details of OSGi modules later. For now, take comfort that we've
+chipped away at the Liferay 6.2 monolith to bring you Liferay 7 as a robust
+system of modules. Next, we'll consider how their delivered to you as a Liferay
+consumer. 
+
+## Core, Apps, and Suites
+
+We've taken Liferay Portal the monolith, broken it down to its parts, decoupled
+them, and re-engineered them into a much more manageable and agile system. Before
+going into details of how we implemented particulars of modularity, let's take a
+look at how Liferay 7's modules are arranged.
+
+![Figure 2: Liferay 7 is composed of the Liferay Core and several suites, each with their own set of application and framework modules.](../../images/core-suites-apps.png)
+
+What does all this mean? Has Liferay Portal simply been repackaged? No;
+it's been redeveloped into independent robust modules that work together to
+bring you a full-fledged portal. And on top of that, we've made the modules
+accessible in aggregations that are easier to release. We'll brief you on them
+now. 
+
+All of the suites run on Liferay Core. The core bootstraps the system, manages
+modules, and registers services. In addition to relying on the core, the suites
+rely on the Liferay Foundation suite for the building blocks familiar to Liferay
+users and developers, such as roles and permissions, users, groups, and user
+groups, sessions, login, and licensing, upgrades, clustering, and DAO, and
+front-end mainstays for themes, CSS, taglibs, and JavaScript. The Foundation
+suite's modules deploy on Liferay Core, as do all of the non-core modules.
+
+And then there are Liferay 7's other new suites that provide modules for the
```

Comment:
Instead of describing them as new suites, you might want to say that Liferay's applications have been reorganized into suites.